src/stories/list.stories.js

```diff
@@ -50,11 +46,33 @@ export const ListA = () => {
 
     return (
         <StoriesContainer>
+            <p>Profile takes key/value - value as string or number only These are dumb componets</p>
             <Card title="Mentor List Example" onEdit={() => console.log("Edit is Clicked!")}>
```

Comment:
It's totally fine to use `Card` here for styling but I'd suggest to omit `onEdit` as this is out of context for this story
```suggestion
            <Card title="Mentor List Example">
```